sdk/common/src/main/java/io/opentelemetry/sdk/common/export/ConfigBuilder.java

```diff
@@ -158,4 +159,91 @@ protected static Double getDoubleProperty(String name, Map<String, String> map)
       return null;
     }
   }
+
+  /**
+   * Get a duration property from the map, {@code null} if it cannot be found or it has a wrong
```

Comment:
Thanks a lot for the help! Configuration is still in flux, and this functionality just moved to a different location. Can you put it there instead?

https://github.com/open-telemetry/opentelemetry-java/blob/master/sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/ConfigProperties.java

Reply:
Done.